presentation/backend/src/presentation-backend/Presentation.ts

```diff
@@ -120,7 +118,7 @@ export class Presentation {
     this._shutdownListener = IModelHost.onBeforeShutdown.addListener(() => Presentation.terminate());
 
     this._rpcImpl = new PresentationRpcImpl({
-      requestTimeout: this._initProps.requestTimeout ?? defaultRequestTimeout,
+      requestTimeout: this._initProps.requestTimeout ?? DEFAULT_REQUEST_TIMEOUT,
```

Comment:
Is it necessary to pass `DEFAULT_REQUEST_TIMEOUT` here? It looks like `PresentationRpcImpl` uses `DEFAULT_REQUEST_TIMEOUT` if props are not provided. Maybe `Presentation.initilize` could check if `this._initProps.requestTimeout` is defined otherwise pass undefined to `PresentationRpcImpl`?

Reply:
Agreed and fixed.